docs/sources/configuration/index.template

```diff
@@ -0,0 +1,98 @@
+---
+description: Describes parameters used to configure Grafana Loki.
+menuTitle: Configuration parameters
+title: Grafana Loki configuration parameters
+weight: 500
+---
+
+# Grafana Loki configuration parameters
+
+{{ .GeneratedFileWarning }}
+
+Grafana Loki is configured in a YAML file (usually referred to as `loki.yaml` )
+which contains information on the Loki server and its individual components,
+depending on which mode Loki is launched in.
+
+Configuration examples can be found in the [Configuration Examples](examples/) document.
+
+## Printing Loki Config At Runtime
```

Comment:
I think we use sentence case for titles

```suggestion
## Printing Loki config at runtime
```

Reply:
See also other headlines in this document

Reply:
👍 sounds good. Will address these comments in a separate PR